### DISCUSSION TITLE
0 query hits (0 already seen), 0 matches, 0 alerts sent

### DISCUSSION THREAD

**Discussion by harshalizode** [jertel/elastalert2#1279] [Q&A] [answered]
Hi guys,
setup of elastalert2

-- logs
INFO:elastalert:Ran Example frequency rule from 2023-09-25 11:54 India Standard Time to 2023-09-25 12:09 India Standard Time: 0 query hits (0 already seen), 0 matches, 0 alerts sent
INFO:elastalert:Example frequency rule range 900
INFO:elastalert:Disabled rules are: []
INFO:elastalert:Sleeping for 299.999781 seconds
INFO:elastalert:Background configuration change check run at 2023-09-25 12:14 India Standard Time
INFO:elastalert:Background alerts thread 0 pending alerts sent at 2023-09-25 12:14 India Standard Time
INFO:elastalert:Queried rule Example frequency rule from 2023-09-25 11:59 India Standard Time to 2023-09-25 12:14 India Standard Time: 0 / 0 hits

-- config.yaml

es_host: 10.1.3.28
es_port: 9200
es_username: root
es_password: pass@123

use_ssl: True
verify_certs: false

rules_folder: example_rules

run_every:
  minutes: 5

buffer_time:
  minutes: 15

writeback_index: elastalert_status
writeback_alias: elastalert_alerts

# If an alert fails for some reason, ElastAlert will retry
# sending the alert until this time period has elapsed
alert_time_limit:
  days: 10

-- rule.yaml
name: Example frequency rule
type: frequency

index: k_resource

num_events: 5

timeframe:
  hours: 1

timestamp_field: timestamp
timestamp_type: unix

filter:
  - term:
      param: "DISK_USAGE"

alert:
  - email
email:
  - abc@gmail.com

smtp_host: 'smpt.gmail.com'
smtp_port: 587
smtp_ssl: true
smtp_auth_file: smtp_auth.yaml
from_addr: 'xyz@gmail.com'

Can you help me to figure out what I am doing wrong.
Thank you

**Comment by jertel** [comment, marked as answer]
Use Kibana to define a query that returns the expected results. Then look at the Kibana _inspection_ popup to find the raw query that was used. Compare that to the query filter and index you are attempting to use in ElastAlert 2.

**Reply by harshalizode** [reply]
Here, in my query range filter is not working, how can I fix this?

Query is: 
POST kpas_resource/_search
{
  "query": {
    "range": {
      "timestamp": {
        "gte": 1695621204,
        "lt": 1695643529
      }
    }
  }
}

Mapping is:
"timestamp": {
          "type": "date",
          "format": "epoch_second"
        }

Document is:
"_source": {
          "code": "tesseurr",
          "timestamp": 1695627480,
          "value": 48.4,
          "zone": "GREEN"
        }

**Comment by harshalizode** [comment]
How can we pass the unix timestamp in string using elastalert2?

"range": {
"timestamp": {
"gte": 1695621204,
"lt": 1695643529
}
}

to

"range": {
"timestamp": {
"gte": "1695621204",
"lt": "1695643529"
}
}
